src/doc/src/faq.md

```diff
@@ -259,3 +259,50 @@ Some issues we've seen historically which can cause crates to get rebuilt are:
 If after trying to debug your issue, however, you're still running into problems
 then feel free to [open an
 issue](https://github.com/rust-lang/cargo/issues/new)!
+
+### What does “version's conflict” mean and how to resolve it?
```

Comment:
I don't know what's going on when my phone's autocorrection. My bad. However, this should be

```suggestion
### What does "version conflict" mean and how to resolve it?
```

And we don't need curly quotes. mdbook will convert them for us. My bad again.

Reply:
Thank you very much for your guidance. This was my mistake, I didn't double check.